src/pieces/commands/assets.py

```diff
@@ -81,96 +81,52 @@ def open_asset(**kwargs):
     print_model_details(asset_dict["name"],asset_dict["created_at"],asset_dict["updated_at"],asset_dict["type"],asset_dict["language"],filepath)
 
 def save_asset(**kwargs):
```

Comment:
might want to adjust this command to something like update Asset transferable, which will refer to the asset core value